mcs/class/corlib/corert/Interop.Number.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+internal static partial class Interop
+{
+    internal unsafe partial class Sys
+    {   
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern unsafe int DoubleToString(double value, byte* format, byte* buffer, int bufferLength);
+    }
+}
```

Comment:
instead of copying the file we should make #if MONO in the corert version (like we did with Interop.RunLoop.cs).

Reply:
yeah probably makes sense